content/docs/atom-guide/dev-guide/shaders/azsl/srg-semantics.md

```diff
@@ -210,12 +211,12 @@ Finally, if we compile to use a register space per SRG, with *--use-spaces*, and
     
     ConstantBuffer<::DrawSrg_SRGConstantsStruct> DrawSrg_SRGConstantBuffer : register(b3, space0);
     
-  
+```
 Note, for each register space, in this case, each SRG, all registers start at 0, BUT resources that belong to `DrawSrg` are bound to register space 0, while resources that belong to `ObjectSrg` are bound to space 1. This ordering of register spaces was determined by the `FrequencyId` member of the SRG Semantics.  
   
 ## ShaderVariantFallback: Defining Who Owns The Bit Array Of Options
 `ShaderVariantFallback` is an optional keyword that marks an SRG as the owner of the array of bits where all Shader Variants will be encoded.  
-Because there can only be one array of bits to encode the Shader Variant Options, only one SRG can bind to an SRG Semantic that uses the `ShaderVariantFallback` keyword.  
-If a shader makes no use of Shader Variant Options, then the `ShaderVariantFallback` is not required to be used.  
+Because there can only be one array of bits to encode the Shader variant options, only one SRG can bind to an SRG Semantic that uses the `ShaderVariantFallback` keyword.  
```

Comment:
```suggestion
Because there can only be one array of bits to encode the shader variant options, only one SRG can bind to an SRG Semantic that uses the `ShaderVariantFallback` keyword.  
```

Reply:
Fixed.